JASP-Engine/JASP/R/descriptives.R

```diff
@@ -808,29 +808,30 @@ Descriptives <- function(jaspResults, dataset, options) {
 
 
 .descriptivesFrequencyPlots <- function(dataset, options, variable) {
+  
   if (options$splitby != "" ) {
     # return a collection
     split <- names(dataset)
 
     plotResult <- createJaspContainer(title=variable)
-    plotResult$dependOn(options="splitby", optionContainsValue=list(variables=variable))
+    plotResult$dependOn(options=c("splitby", "binWidthType"), optionContainsValue=list(variables=variable))
 
     for (l in split) {
-      plotResult[[l]] <- .descriptivesFrequencyPlots_SubFunc(dataset=dataset[[l]], variable=variable, width=options$plotWidth, height=options$plotHeight, displayDensity = options$distPlotDensity, title = l)
+      plotResult[[l]] <- .descriptivesFrequencyPlots_SubFunc(dataset=dataset[[l]], variable=variable, width=options$plotWidth, height=options$plotHeight, displayDensity = options$distPlotDensity, rugs = options$distPlotRug, title = l, binWidthType = options$binWidthType)
       plotResult[[l]]$dependOn(optionsFromObject=plotResult)
     }
 
     return(plotResult)
   } else {
     column <- dataset[[.v(variable)]]
-    aPlot <- .descriptivesFrequencyPlots_SubFunc(dataset=dataset, variable=variable, width=options$plotWidth, height=options$plotHeight, displayDensity = options$distPlotDensity, title = variable)
-    aPlot$dependOn(options="splitby", optionContainsValue=list(variables=variable))
+    aPlot <- .descriptivesFrequencyPlots_SubFunc(dataset=dataset, variable=variable, width=options$plotWidth, height=options$plotHeight, displayDensity = options$distPlotDensity, rugs = options$distPlotRug, title = variable, binWidthType = options$binWidthType)
+    aPlot$dependOn(options=c("splitby", "binWidthType"), optionContainsValue=list(variables=variable))
```

Comment:
Consider e.g., https://legacy.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds

So it's good that you add a space before and after each `=` but try to be consistent within the function call. So either add a space before and after each `=` or nowhere.